scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if doIsAzureInstalled
```

Comment:
if the command to test this is only as simple as `command -v az &> /dev/null` you could directly have it here instead of adding an extra function with 9 lines of code that does nothing else.
However, nothing wrong to do it this way.